app/controllers/users/registrations_controller.rb

```diff
@@ -28,9 +47,11 @@ def update
    end
 
   # DELETE /resource
-  # def destroy
-  #   super
-  # end
+   def destroy
+    puts "###########"
```

Comment:
remember delete this if you don't need it (puts "###")

Reply:
sorry for that i forget delete 😿